userspace/libsinsp/container.cpp

```diff
@@ -736,16 +795,9 @@ bool sinsp_container_manager::parse_docker(sinsp_container_info* container)
 			break;
 	}
 
-	size_t pos = json.find("{");
-	if(pos == string::npos)
-	{
-		ASSERT(false);
-		return false;
-	}
-
 	Json::Value root;
 	Json::Reader reader;
-	bool parsingSuccessful = reader.parse(json.substr(pos), root);
+	bool parsingSuccessful = reader.parse(json, root);
```

Comment:
I guess this searching for the '{' is no longer needed, as you're expecting libcurl to properly return errors? Is it still not required on windows?

Reply:
I'm not sure I understand what you mean.
For windows, I'm doing it directly inside `get_docker`.
When using libcurl is not required because `json` will already contain only the response json, not the full HTTP response as before, or will fail with errors.